storybook/tsconfig.storybook.json

```diff
@@ -1,5 +1,5 @@
 {
-  "extends": "../../tsconfig.json",
+  "extends": "../tsconfig.json",
```

Comment:
I've moved all storybook related config to `/storybook`. The alternative was to duplicate this config in each new draft package and that felt icky to me.